vector/src/main/java/im/vector/app/features/roomprofile/members/RoomMemberListFragment.kt

```diff
@@ -131,5 +137,6 @@ class RoomMemberListFragment @Inject constructor(
             roomSettingsToolbarTitleView.text = it.displayName
             avatarRenderer.render(it.toMatrixItem(), roomSettingsToolbarAvatarImageView)
         }
+        inviteUsersButton.isVisible = state.actionsPermissions.canInvite
```

Comment:
This is not really the room summary rendering.
I would have add this line of code at the end of `invalidate()` or by creating a new dedicated method.
Also what will happen if the state is updated while scrolling? (this is a minor bug, though)

Reply:
Done. I think RecyclerView dragging event will be triggered quickly and probably it won't be visible at all.